utils/telegraf/Makefile

```diff
@@ -0,0 +1,41 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=telegraf
+PKG_VERSION:=1.19.1
+PKG_RELEASE:=1
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/influxdata/telegraf/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=cec43bb0acfff8b4c963ffec6e3eab44ffb52c8f34e6a697207977cfd05882aa
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Jonathan Pagel <jonny_tischbein@systemli.org>
+PKG_LICENSE:=MIT
+
+include $(INCLUDE_DIR)/package.mk
+include $(TOPDIR)/feeds/packages/lang/golang/golang-values.mk
+
+MAKE_VARS = \
+	GOARCH=$(GO_ARCH)
+
+define Package/telegraf
+	SECTION:=utils
+	CATEGORY:=Utilities
+	TITLE:=Telegraf
+	PKGARCH:=all
```

Comment:
here you wrote all. maybe that is a mistake? and also the DEPENDS should exclude targets with no go support (maybe, I'm not sure)... ;)

Reply:
Should I remove `PKGARCH` and only use `DEPENDS` or both with `$(GO_ARCH_DEPENDS)` ?

Reply:
I think I would do it like that (but not sure if it fixes your issue).

Reply:
I did fix it and the CI test was successfull. Should I resolve this ?